src/dotnet/UI.Blazor.App/Pages/EmbeddedChatPage.razor

```diff
@@ -102,22 +99,16 @@
 
                 _ = InvokeAsync(async () => {
                     // Set new ChatSid
-                    ChatSid = newChat.Id;
-                    await SelectChat(chatSid);
+                    await SelectChat(newChat.Id);
```

Comment:
no need for async/await?

Reply:
I did not get what you mean. Can you restate, please?